doc/README.fr.md

```diff
@@ -553,7 +553,7 @@ Liste à deux dimensions de sons (0-63) listés par leur numéro de canal.
 Met les listes de sons (0-63) de tous les canaux. Si une liste vide est passée, ce canal n’est pas utilisé.<br>
 Par exemple `pyxel.music(0).set([0, 1], [2, 3], [4], [])`
 
-### Advanced APIs
+### APIs avancées
 
 Pyxel a des "APIs avancées" qui ne sont pas présentées ici car elles peuvent amener de la confusion ou qu’elles nécessitent des connaissances spécifiques pour être utilisées.
```

Comment:
Dans l’autre PR j’ai remplacé quelques «les API» (sans s pour un pluriel d’abréviation 😉) par «l’API».
Même si la source en anglais utilise «APIs» (dans le sens _functions that are part of the API_ je pense), les fonctions utilisables constituent _une_ API à mon sens, l’interface de programmation de Pyxel. Donc je propose de changer `API avancée` dans le titre et `Pyxel a une API avancée qui n’est pas présentée etc` dans le texte.

Reply:
Je vais changer ça, ça sera plus cohérent et plus correct :)